src/components/paragraph/rows/three-column.tsx

```diff
@@ -44,7 +44,9 @@ const ThreeColumn = async ({items, fullWidth = true, config}: Props) => {
       })}
     >
       <div
-        className="centered flex flex-col justify-between gap-90 *:mx-auto *:w-full md:flex-row md:flex-wrap *:md:w-[calc(50%_-_5rem)] lg:flex-nowrap *:lg:w-[calc(33.3%_-_5rem)]"
+        className={clsx(
+          "centered flex w-full flex-col justify-between gap-90 *:mx-auto *:w-full md:w-[124rem] md:flex-row md:flex-wrap *:md:w-[calc(50%_-_5rem)] lg:flex-nowrap *:lg:w-[calc(33.3%_-_5rem)]"
```

Comment:
Move the `*:` styles down directly on the `<OneColumn>` component. Those styles should get applied directly on the appropriate elements.